### DISCUSSION TITLE
404 Error - Second Repo not displaying Github Pages

### DISCUSSION THREAD

**Discussion by anilgeorge04** [mmistakes/minimal-mistakes#2748] [Q&A]
Thanks for this amazing theme. I am having a great time building my site out because of the rich documentation and community.
I have read the theme's documentation and gone over the Jekyll documentation.

### My Issue
I am getting a 404 error while accessing a Github Page outside of my primary repo [anilgeorge04.github.io/minimal-cave](https://github.com/anilgeorge04/minimal-cave) which uses Minimal Mistakes.

I have another repo [anilgeorge04/firststep](https://github.com/anilgeorge04/firststep) where I've setup Github Pages to publish from the main branch.

![image](https://user-images.githubusercontent.com/62422148/101009778-1d664c80-3588-11eb-88a3-bec92cafb680.png)

The page seems to be loading fine according to Github Pages with a valid URL.

Because the [page](https://minimalcave.com/firststep/) was throwing me a 404 error, I added a [_config.yml](https://github.com/anilgeorge04/firststep/blob/main/_config.yml) file in the firststep repo based on the discussion on similar topics at Github Community. It still shows up my [404 page](https://github.com/anilgeorge04/minimal-cave/blob/master/docs/_pages/404.md) configured on the primary repo.

### More Details
My custom domain minimalcave.com is mapped (using CNAME) with [anilgeorge04.github.io/minimal-cave](https://github.com/anilgeorge04/minimal-cave) repo.

The [minimalcave.com/firststep/](https://minimalcave.com/firststep/) link was working just fine at the start, until I configured my 404.md page on the primary repo.

### My Query
I think I'm missing something obvious here perhaps.
How can I successfully continue to host my other Github Pages, from other repos, using my custom domain followed by repo-name, without throwing up a 404 error?
Is there any exclusion in the _config file I can add? Or any inclusions in the front matter for my 404 page?

I'm learning Jekyll and web programming along the way, so I'm finding it difficult to troubleshoot further. Github Pages documentation also isn't helping overcome this specific issue.

Thanks for taking the time.

**Comment by mmistakes** [comment]
Doesn’t seem to be theme or Jekyll related. I think your issue is all GitHub Pages and how you’re trying to setup multiple repos. 

I’ve never gone the custom domain path with GH Pages so I won’t be any help. I’d post over on the Jekyll forums instead, maybe you’ll have better luck.

https://talk.jekyllrb.com/

**Comment by anilgeorge04** [comment]
Yeah I figured this is more of a Github Page/Hosting/Domain mapping issue. I wanted to make sure it wasn't related to how the 404 page is configured. 

Thanks Michael. I've posted on the Jekyll forum. [Here](https://talk.jekyllrb.com/t/404-error-second-repo-not-displaying-github-pages/5311) is the post for future reference, if anyone else faces a similar issue.
